lua/lib/physrep_register_replicant.lua

```diff
@@ -5,6 +5,12 @@
 -- register themselves. The node, in return, sends back a list of "potential"
 -- nodes that the replicant can connect to to pull and apply physical logs.
 local function main(dbname, hostname, lsn, source_dbname, source_hosts)
+
+    print(
```

Comment:
nitpick : maybe useful to have physrep_fanout and physrep_max_candidates as well here ?